h5io/_h5io.py

```diff
@@ -335,6 +343,7 @@ def _triage_write(
             where + ".csr_matrix_shape",
             cleanup_data=cleanup_data,
             slash=slash,
+            use_state=use_state,
```

Comment:
Is there a case where the `shape` are required to be an object which needs `__getstate__()` to be stored in the HDF5 file?